### PR TITLE
Add role to cluster task for create_rds playbook.

### PR DESCRIPTION
**Pull request by HassanJaveed84**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: playbooks/create_rds.yml [hunk 1]
```diff
@@ -72,5 +72,14 @@
...
     delay: 60
 
+  - name: Add role to DB cluster
+    command: "aws rds add-role-to-db-cluster
+              --db-cluster-identifier {{ cluster_name }}
+              --role-arn {{ cluster_role_arn }}
+              "
+    register: result
+    changed_when: "result.rc == 0"
+    when: cluster_name is defined and cluster_role_arn is defined
```
Comment (adzuci) [inline playbooks/create_rds.yml:84]: After we run this the first time, should we add `and ('DBInstanceAlreadyExists' not in result.stderr)` to it or can we confirm that you think this command is idempotent?

**Review by adzuci — APPROVED** [review]
:shipit:

File: playbooks/create_rds.yml (hunk 1 above)
Comment (HassanJaveed84) [reply playbooks/create_rds.yml:84]: I can check that.

File: playbooks/create_rds.yml [hunk 2]
```diff
@@ -72,5 +72,16 @@
...
     delay: 60
 
+  - name: Add role to DB cluster
+    command: "aws rds add-role-to-db-cluster
+              --db-cluster-identifier {{ cluster_name }}
+              --role-arn {{ cluster_role_arn }}
+              "
+    register: add_role_result
+    failed_when: >
+      add_role_result.rc !=0 and ('DBClusterRoleAlreadyExists' not in add_role_result.stderr)
```
Comment (HassanJaveed84) [inline playbooks/create_rds.yml:82]: @adzuci I've added this so it will not fail if the role already exists.

**Review by adzuci — APPROVED** [review]

**Pull request merged**